test/e2e/windows/hybrid_network.go

```diff
@@ -79,6 +80,58 @@ var _ = SIGDescribe("Hybrid cluster network", func() {
 
 		})
 
+		ginkgo.It("should have stable networking for Linux and Windows with services", func() {
+			serviceName := "stable-networking"
+			ginkgo.By("creating service " + serviceName + " with type=ClusterIP in namespace " + f.Namespace.Name)
+			jig := e2eservice.NewTestJig(f.ClientSet, f.Namespace.Name, serviceName)
+
+			svc, err := jig.CreateTCPService(func(svc *v1.Service) {
+				svc.Spec.Type = v1.ServiceTypeClusterIP
+			})
+			framework.ExpectNoError(err)
+
+			ginkgo.By("creating some linux pods in the service")
+			var linuxPod *v1.Pod
+			for i := 0; i < 2; i++ {
+				ginkgo.By("creating a linux pod and waiting for it to be running")
+				linuxPod = createTestPod(f, linuxBusyBoxImage, linuxOS)
+				linuxPod.ObjectMeta.Labels = svc.Labels
+				linuxPod = f.PodClient().CreateSync(linuxPod)
+			}
+
+			ginkgo.By("creating some Windows pods in the service")
+			var windowsPod *v1.Pod
+			for i := 0; i < 2; i++ {
```

Comment:
do we need to create 2 pods? 
does it matters for this test since only the last one is being used.

Reply:
if you have more that one pod you should check that all pods are polled during the test, the other networking tests has logic for that ...

Reply:
I looked through the networking tests for this. I didn't see any explicit check that ever pod was hit (maybe I missed it) but I did find some checks where the pods are added then deleted and this seems to indicate that they were both tested:

https://github.com/kubernetes/kubernetes/blob/469c4c4a30ab57540425e739215f4ceaf9a30682/test/e2e/network/service.go#L812-L831

Is this what you are referring to?

Reply:
if you go deep into this function

https://github.com/kubernetes/kubernetes/blob/469c4c4a30ab57540425e739215f4ceaf9a30682/test/e2e/network/networking.go#L154

you can see it receives the endpoint as parameter and inside it checks that all the endpoints are polled